backend/de.metas.business/src/main/java/de/metas/shipping/process/C_Order_AddTo_M_ShipperTransportation.java

```diff
@@ -53,15 +59,24 @@ public ProcessPreconditionsResolution checkPreconditionsApplicable(@NonNull fina
 	{
 		if (context.isNoSelection())
 		{
-			return ProcessPreconditionsResolution.rejectBecauseNoSelection();
+			return ProcessPreconditionsResolution.rejectBecauseNotSingleSelection();
 		}
 
-		final OrderId orderId = OrderId.ofRepoId(context.getSingleSelectedRecordId());
-		final org.compiere.model.I_C_Order order = orderBL.getById(orderId);
-		if (orderBL.isRequisition(order))
+		final IQueryFilter<I_C_Order> queryFilter = context.getQueryFilter(I_C_Order.class);
+		final List<I_C_Order> selectedOrders = orderBL.getByQueryFilter(queryFilter);
```

Comment:
what happens if user opens the orders window for all orders (4000), presses Select All from all pages?
how much will take to compute the preconditions?